dat/outfits/launchers/zalek_bomber_drone_fighter_dock.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<outfit name="Za'lek Bomber Drone Fighter Dock">
+ <general>
+  <slot>weapon</slot>
+  <size>large</size>
+  <license>Light Combat Vessel License</license>
+  <rarity>2</rarity>
+  <mass>200</mass>
+  <price>1200000</price>
+  <gfx_store>zalek_drone_bomber_fighter_swarm.png</gfx_store>
```

Comment:
You didn't seem to include this file. If you do it should be pushed to naev-production-artwork instead unless you want me to do it for you.